extensions/podman/src/podman-remote-connections.ts

```diff
@@ -169,7 +169,11 @@ export class PodmanRemoteConnections {
       const sshTunnel = this.createTunnel(host, port, username, privateKey, remotePath, localPath);
 
       // connect the tunnel
-      sshTunnel.connect();
+      try {
+        sshTunnel.connect();
+      } catch (error) {
+        console.error(`Error connecting to Podman SSH Remote: ${error}`);
```

Comment:
```suggestion
        console.error(`Error connecting to Podman SSH Remote`, error);
```

I think we may lose some traces if we stringify the error